crates/ruff_python_formatter/src/expression/string.rs

```diff
@@ -323,17 +323,25 @@ impl StringPart {
         self,
         quoting: Quoting,
         locator: &'a Locator,
-        quote_style: QuoteStyle,
+        configured_style: QuoteStyle,
     ) -> NormalizedString<'a> {
+        // Per PEP 8 and PEP 257, always prefer double quotes for docstrings and triple-quoted
+        // strings. (We assume docstrings are always triple-quoted.)
+        let preferred_style = if self.quotes.triple {
```

Comment:
Nit: The variable here is called `preferred_style` but it is called `configured_style` in `peferred_quotes_raw` and `preferred_quotes`. 

I would be okay calling it `configured_style` or maybe `default_style` to make it clear it's what we would default to.

Reply:
Yeah I went back and forth on these a few times :joy: